electron/src/main.ts

```diff
@@ -212,14 +212,14 @@ const bindIpcEvents = (): void => {
   ipcMain.handle(EVENT_TYPE.ACTION.GET_OG_DATA, (_event, url) => getOpenGraphDataAsync(url));
 
   ipcMain.on(EVENT_TYPE.ACTION.CHANGE_DOWNLOAD_LOCATION, (_event, downloadPath?: string) => {
-    if (EnvironmentUtil.platform.IS_WINDOWS) {
-      if (downloadPath) {
-        fs.ensureDirSync(appHomePath(downloadPath));
-      }
-      //save the downloadPath locally
-      settings.save(SettingsType.DOWNLOAD_PATH, downloadPath);
-      settings.persistToFile();
+    // if (EnvironmentUtil.platform.IS_WINDOWS) {
+    if (downloadPath) {
+      fs.ensureDirSync(appHomePath(downloadPath));
     }
+    //save the downloadPath locally
+    settings.save(SettingsType.DOWNLOAD_PATH, downloadPath);
+    settings.persistToFile();
+    // }
```

Comment:
are you sure about those changes?

Reply:
ah sorry these were unrelated and floating on my branch apparently.

Reply:
from my last testing of download path